src/test/java/com/adyen/TransfersTest.java

```diff
@@ -0,0 +1,45 @@
+package com.adyen;
+
+import com.adyen.model.transfer.*;
+import com.adyen.service.Transactions;
+import com.adyen.service.Transfers;
```

Comment:
I think that for a small service like this we don't need two classes, how about combining all endpoints into a single class?

Reply:
Ola! Devil's advocate here 😈 !

@AlexandrosMor  is actually following our API documentation [here](https://docs.adyen.com/api-explorer/transfers/3/overview)

Do we follow suit or deviate?

Reply:
It's a bit overkill, but we can have two small classes. Still, they should be combined via single parent class. We don't have two separate `com.adyen.service.Transactions ` and `com.adyen.service.Transfers` services, it's only one **Transfers API**.